web/frontend/components/CollapseTriangle.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
If anyone with a keener eye has opinions about a more shapely triangle, I'm not attached to this one.